core/stores/kv/store.go

```diff
@@ -110,7 +110,9 @@ type (
 		Ttl(key string) (int, error)
 		TtlCtx(ctx context.Context, key string) (int, error)
 		Zadd(key string, score int64, value string) (bool, error)
+		ZaddFloat64(key string, score float64, value string) (bool, error)
```

Comment:
I think ZaddFloat & ZaddFloatCtx are better, like ParseFloat & FormatFloat in stdlib.

Reply:
done